src/components/layout/Header.tsx

```diff
@@ -79,7 +79,7 @@ const Header: React.FC = () => (
           Connect to Wallet
         </a>
       </nav>
-      <div class="header-title">
+      <div className="header-title">
```

Comment:
Unrelatead change

<a href="https://gitme.me/image?url=https%3A%2F%2Fmedia1.giphy.com%2Fmedia%2FftviKzVUcnct2%2Fgiphy.gif&token=unrelated" data-gitmeme-token="unrelated"><img src="https://media1.giphy.com/media/ftviKzVUcnct2/giphy.gif" title="Created by gitme.me with /unrelated"/></a>

Reply:
You got me, but not so unrelated :) 

This change it's because it breaks the compilation in travis because of that. Would you rather create a PR only to fix that?

Reply:
No, it's fine but pointing it out.